src/sentry/sentry_metrics/indexer/cloudspanner_model.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+from dataclasses import dataclass
+from typing import Sequence
+
+_TIMESTAMP_TZ_FORMAT = "%Y-%m-%d %H:%M:%S %Z"
+
+@dataclass
+class SpannerIndexerModel:
+    id: int
+    string: str
+    organization_id: int
+    date_added: datetime.datetime
+    last_seen: datetime.datetime
+    retention_days: int
```

Comment:
I would suggest we add an additional (Numeric) column to store the decoded unsigned integer we would store in clickhouse. This will make the troubleshooting a lot easier as the id we would be storing in the primary key is tweaked quite a bit for proper partitioning and because of the Signed integer issue (clickhouse is unsigned).

We do not need to expose this column but we should have it on the table.